assignment02/js/myScript2.js

```diff
@@ -0,0 +1,36 @@
+$(document).ready(function(){
+    var female1 = prompt("Please enter a female name (1 of 2).");

```

Comment:
I like that you prompt and then assign back to the HTML with jQuery right away to keep things tidy. Great numbering and grouping of like fields.